sources/sos-report/leapp.py

```diff
@@ -0,0 +1,18 @@
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class Leapp(Plugin, RedHatPlugin):
+    """
+    This plugin is used to gather all information necessary for reporting
+    problems in regards to the leapp application.
+    """
+
+    name = 'leapp'
```

Comment:
```suggestion
    plugin_name = 'leapp'
```

Reply:
https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin they don't say 'plugin_name' for what ever reason.

Reply:
last edit is 2015, so I guess they changed it and didn't update docs, existing plugins use `plugin_name`